### ISSUE TITLE
Additional/custom fields for game details

### ISSUE THREAD

**Issue by By6r9** [JosefNemec/Playnite#286]
Game details window lacks some fields. 

For LaunchBox parity:
Region, Version, Age Rating, Added, Modified, Last Played, Play Count, Source, Status, Play Mode, Series, Alternate Name
Game Manual Path, Game Music Path, Game Video Path 

For MAME/Arcade emulators:
Clone of, Romset, Status, Graphics status, Sound status, Mechanical machine, Rotation/Screen Orientation, Display Count, Display Type, AltRomname, AltTilte, Buttons (and some more...)

**Comment by Snorrp** [comment]
Yuss, how far along would you say these are.

**Comment by JosefNemec** [comment]
@Snorrp Not sooner than 4.0, which is at least a month away.

**Comment by Snorrp** [comment]
Ah that's not too bad, you work at a good pace.

**Comment by outsider7a** [comment]
Maybe it could exist an optional field to set the shortcut name? Just to avoid problems when a game's name have special characters like ":"

**Comment by JosefNemec** [comment]
What problem is there with shortcuts? I don't think it's a worth it to create separate field just for shortcut name, but we should fix any issues with shortcut creation if there are any.

**Comment by outsider7a** [comment]
Yes i agree. The problem is when a game's name has special characters, the shortcut changes it for another character. For example a shortcut for "Half-Life 2: Episode One" will result in "Half-Life 2_ Episode One".
I didn't wanted to create another issue for this so i thought it could be "fixed" with a custom field.

**Comment by JosefNemec** [comment]
Please open new issue for it, it will be easy to fix.

**Comment by JosefNemec** [comment]
I'm going to add following fields as part of this issue:
    Added
    Modified
    Last Played
    Play Count
    Series
    Version
    Age Rating
    Region
    Version

These will be added as part of #273:
    Rating
    Critic Rating

And "Status" as part of #281

Full support for custom fields will be done as part of #502

"Game Manual Path" will be done in #503 (we also need an option to actually utilize it in Playnite from game menus)

"Game Music Path" and "Game Video Path" will be done as part of #358

And finally "Play Mode" can be supplied with tags based on what I saw how it's used in LaunchBox.

I'm going to leave out all fields for MAME/Arcade because they feel way too specific and it doesn't make much sense to include them for now since we don't have any specific integration for MAME or other arcade emulators.

**Comment by JosefNemec** [comment]
I forgot about "Source", what is it used for?

**Comment by By6r9** [comment]
I realized that I missed another important field - Disc number for games released on multiple discs. I guess it could be added with #159, but it is worth mentioning, so you can plan Game Details layout ahead.
There are also 5¼-inch floppies, with Side A and Side B. 
TOSEC naming scheme mix both those things, so you get things like:
```
Ultima IV - Quest of the Avatar (1986)(Origin Systems)(Disk 1 of 2 Side A)[cr ECA]
Ultima IV - Quest of the Avatar (1986)(Origin Systems)(Disk 1 of 2 Side B)
Ultima IV - Quest of the Avatar (1986)(Origin Systems)(Disk 2 of 2 Side A)
Ultima IV - Quest of the Avatar (1986)(Origin Systems)(Disk 2 of 2 Side B)
